lib/internal/debugger/inspect_repl.js

```diff
@@ -575,8 +575,16 @@ function createRepl(inspector) {
     if (input === '\n') return lastCommand;
     // Add parentheses: exec process.title => exec("process.title");
     const match = RegExpPrototypeExec(/^\s*(?:exec|p)\s+([^\n]*)/, input);
+    input = match ? match[1] : input;
+
+    // Add parentheses to make sure input is parsed as expression
+    if (RegExpPrototypeExec(/^\s*{/, input) !== null &&
```

Comment:
I think we also should append `lib/internal/debugger/inspect_repl.js` to this comment:
https://github.com/nodejs/node/blob/31156a7e9dfc2d98175b10764008aaf23d9f4b6e/lib/repl.js#L417-L421